CHANGELOG.md

```diff
@@ -1,25 +1,45 @@
-# Unreleased
+# Unreleased: 12.0.0 (Breaking release)
 
-## Features
+## Breaking changes 
 
-### Preserve query strings
+This release ensures the GOV.UK Prototype Kit reflects the latest release of the GOV.UK Frontend, v4.0.0.
 
-We now preserve URL query strings when redirecting POST requests to GET requests.
+### Update to GOV.UK Frontend v4.0.0
 
-This means if you have a query like `/link/to/something?query=true&hello=world` on your POST form action, and you submit the form, the URL query string will be present in the redirected URL.
+The new release of GOV.UK Frontend contains:
+- an iteration to the accordion component
+- guidance about other ‘breaking’ changes you should make to improve your service
 
-This is useful when:
+Check the [GOV.UK Frontend release notes](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0) for changes you may need to make to ensure your prototype works.
 
-- You’re using the query string to set flash messages or return paths
-- You want to rely on explicitly what’s in the query string for a specific page, rather than saved data.
+This change was added in [pull request #1195: Update the GOV.UK Prototype Kit to use GOV.UK Frontend v4.0.0](https://github.com/alphagov/govuk-prototype-kit/pull/1195).
 
-Thanks to @edwardhorsford.
+## New features 
+
+### Preserve query strings when redirecting POSTs to GETs
+
+The GOV.UK Prototype Kit now preserves URL query strings when redirecting POST requests to GET requests.
+
+This means if you have a query like `/link/to/something?query=true&hello=world` on your POST form action, and you submit the form, the URL query string will be present in the redirected URL. 
+
+This feature is useful when you:
+
+- use the query string to set flash messages or return paths
+- want to use the values in the query string for a specific page, rather than saved data
+
+Thanks to [@edwardhorsford](https://github.com/edwardhorsford) for contributing this issue and its solution.
+
+This was added in [pull request #1120: Preserve query string when redirecting POSTs to GETs](https://github.com/alphagov/govuk-prototype-kit/pull/1120).
 
-- [Pull request #1120: Preserve query string when redirecting POSTs to GETs](https://github.com/alphagov/govuk-prototype-kit/pull/1120)
 
 ## Fixes
 
-[Pull request #1155: Replace keypather with lodash.get](https://github.com/alphagov/govuk-prototype-kit/pull/1155)
+- [pull request #1155: Replace `keypather` package with `lodash.get`](https://github.com/alphagov/govuk-prototype-kit/pull/1155)
+
+`keypather` and `lodash.get` are both packages that are used to get deep values within objects.
```

Comment:
My bad again, sorry, since I'm responsible for this content's inclusion! We can probably cut line 39. It's probably enough that we've just added the word 'package' to the PR title, to clarify what these things are. Telling users what these packages _do_ feels outside of our remit, like if we explained to users how to use HTML.